paasta_itests/steps/paasta_metastatus_steps.py

```diff
@@ -57,12 +57,12 @@ def marathon_task_is_ready(context, app_id):
     marathon_tools.wait_for_app_to_launch_tasks(context.marathon_client, app_id, 1)
 
 
-@then(u'paasta_metastatus exits with return code "{expected_return_code}" and output "{expected_output}"')
-def check_metastatus_return_code(context, expected_return_code, expected_output):
+@then(u'paasta_metastatus {flags} exits with return code "{expected_return_code}" and output "{expected_output}"')
+def check_metastatus_return_code(context, flags, expected_return_code, expected_output):
```

Comment:
can you check what happens when ``flags`` is empty?

```Then paasta_metastatus exits with..```

Will ``flags`` just be an empty string?